core/manifest/ConfigStaticManifest.php

```diff
@@ -183,17 +183,15 @@ function getStatics() {
 	}
 
 	/**
-	 * Get the next token to process, incrementing the pointer
+	 * Get the next token to process, incrementing the pointer, skips any whitespace tokens
 	 *
-	 * @param bool $ignoreWhitespace - if true will skip any whitespace tokens & only return non-whitespace ones
 	 * @return null | mixed - Either the next token or null if there isn't one
 	 */
-	protected function next($ignoreWhitespace = true) {
```

Comment:
You don't need to close this PR, you just need to undo this change and push it up (preferably squashed).

Unfortunately SemVer does not make a distinction between APIs we think aren't used and ones we think are.